### PR TITLE
Fix error.html template scripts refs

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: api/templates/error.html [hunk 1]
```diff
@@ -6,10 +6,10 @@
     <link rel="stylesheet" href="/static/css/bootstrap-3.3.5.min.css">
     <link rel="stylesheet" href="/static/css/jasny-bootstrap.min.css" rel="stylesheet">
 
-    <script src="/static/js/react-with-addons-0.13.3.min.js"></script>
```
Comment (jwilander) [inline api/templates/error.html]: Pretty sure we need react-with-addons unless it's all included with 0.14 now

**Comment by jwilander** [comment]
I believe this is just updating the error.html refs to match those in head.html?

**Comment by coreyhulen** [comment]
There is also https://mattermost.atlassian.net/browse/PLT-1808 assigned to @crspeller.  Not sure what the best solution is here.

**Comment by jwilander** [comment]
There's a bunch of extra stuff in head.html that's channel.html specific, although with backstage coming it won't be just for channel.html. Overall solution might be better to modularize head.html a bit so that only needed parts included on the pages that need them

**Comment by enahum** [comment]
I leave it up to you guys

**Comment by crspeller** [comment]
I will have to fix this with my react-router stuff. Going to close this. Thanks @enahum!

**Comment by enahum** [comment]
no worries

**Pull request closed without merging**